lambda/health_package/demo/set_alarm_state.py

```diff
@@ -51,12 +51,12 @@ def toggle_alarm_state(alarm_name, region="eu-west-2"):
         # Set alarm state from Alarm to OK
         print(f"Alarm for {str(alarm_name)} is currently set to {state}")
         set_alarm_state("OK", str(alarm_name), region)
-        print(f"Setting state from ALARM -> OK")
+        print("Setting state from ALARM -> OK")
```

Comment:
perhaps these are things we should `LOG` in `set_alarm_state`?

Reply:
I've changed all the print statements to LOG.